src/lib/Controller/api/supervisor.c

```diff
@@ -1130,10 +1130,16 @@ void wb_supervisor_movie_stop_recording() {
 }
 
 bool wb_supervisor_movie_is_ready() {
+  if (!robot_check_supervisor("wb_supervisor_movie_is_ready"))
```

Comment:
I would use the `__FUNCTION__` macro in new code.

Reply:
As the use of this macro was introduced on develop, this should be done when merging master into develop and not here.